packages/plugins/i18n/admin/src/components/BulkLocaleActionModal.tsx

```diff
@@ -0,0 +1,258 @@
+import * as React from 'react';
+
+import { Table, useTable } from '@strapi/admin/strapi-admin';
+import { Box, Typography, IconButton, Flex, Tooltip } from '@strapi/design-system';
+import { Pencil, CheckCircle, CrossCircle, ArrowsCounterClockwise } from '@strapi/icons';
+import { DocumentStatus } from '@strapi/plugin-content-manager/strapi-admin';
+import { Modules } from '@strapi/types';
+import { stringify } from 'qs';
+import { type MessageDescriptor, useIntl } from 'react-intl';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Locale } from '../../../shared/contracts/locales';
+import { getTranslation } from '../utils/getTranslation';
+
+import { LocaleStatus } from './CMHeaderActions';
+
+type status = Modules.Documents.Params.PublicationStatus.Kind | 'modified';
```

Comment:
```suggestion
type Status = Modules.Documents.Params.PublicationStatus.Kind | 'modified';
```

Types should always be capitalised 👍🏼